README.md

```diff
@@ -92,6 +92,7 @@ This file contains a number of front-end interview questions that can be used wh
 * How is responsive design different from adaptive design?
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
 * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
+* How to create an Up/Down arrow using CSS Styles. 
```

Comment:
I know the technique you're referencing but there are a lot of ways to create shapes in HTML and CSS (not just using borders). That said, SVG seems to be the preferred method over custom fonts, pseudo elements or CSS hacks; It's more flexible for animation and was, essentially, made for this kind of thing (ie. vector graphics). 

How about, instead, we ask something like:
> How would you create a non-[rectilinear](http://en.wikipedia.org/wiki/Rectilinear_polygon) shape for a website? Are there any advantages or disadvantages to your approach?

**Rectilinear** is probably an unfamiliar term for many people but the idea is that you want someone to think *outside the box* (ie. shapes not composed of squares or rectangles). I'm open to other wording if anyone else wants to take a crack at it.